sycl/test-e2e/DeprecatedFeatures/deprecated_removed_sycl_device_filter.cpp

```diff
@@ -2,12 +2,13 @@
 // RUN: env SYCL_DEVICE_FILTER='*' %{run-unfiltered-devices} %t.out &> %t.log
 // RUN: FileCheck %s < %t.log
 //
-// CHECK:      WARNING: The enviroment variable SYCL_DEVICE_FILTER is deprecated.
+// CHECK:      WARNING: The enviroment variable SYCL_DEVICE_FILTER is removed.
```

Comment:
Where does this warning come from? Originally it came from sycl/source/detail/config.hpp in the part that is now masked out by the preview macro, but I would expect it to still report "deprecated" in this test in its current form. I have no idea what it would say if you set `-fpreview-breaking-changes` in line 2, but I suspect it won't be this.